kornia/utils/grid.py

```diff
@@ -37,3 +37,36 @@ def create_meshgrid(
     base_grid: torch.Tensor = torch.stack(
         torch.meshgrid([xs, ys])).transpose(1, 2)  # 2xHxW
     return torch.unsqueeze(base_grid, dim=0).permute(0, 2, 3, 1)  # 1xHxWx2
+
+
+def create_meshgrid3d(
+        depth: int,
+        height: int,
+        width: int,
+        normalized_coordinates: Optional[bool] = True) -> torch.Tensor:
```

Comment:
this should be `normalized_coordinates: bool = True`. I think Optional is for None values.